electrical/README.MD

```diff
@@ -1,33 +0,0 @@
-# JPL Open Source Rover Project
-The JPL Open Source Rover is an open source, build it yourself, scaled down version of the 6 wheel rover design that JPL uses to explore the surface of Mars. The Open Source Rover is designed almost entirely out of consumer off the shelf (COTS) parts. This project is intended to be a teaching and learning experience for those who want to get involved in mechanical engineering, software, electronics, or robotics.
```

Comment:
This readme should actually be kept. We'll handle moving this content elsewhere in another PR

Reply:
The original readme has been restored.